src/inverseRobot.js

```diff
@@ -23,8 +23,23 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const newObj = {};
+  const values = Object.values(robot);
+  const keys = Object.keys(robot);
+
+  values.reverse();
+  keys.reverse();
```

Comment:
<img width="560" alt="Screenshot 2020-08-05 at 14 37 30" src="https://user-images.githubusercontent.com/35385361/89408544-34f00e80-d729-11ea-9483-c80503b7b5c4.png">
no need to change the order of fields inside an object. you need to take a key and swap  it with value, they in the same order as they were before